octoprint_slic3r/profile.py

```diff
@@ -38,148 +38,170 @@ class SeamPositions(object):
-    support_material_interface_layers=3,
-    support_material_interface_spacing=0,
-    raft_layers=0,
-    support_material_enforce_layers=0,
-    dont_support_bridges=True,
-
-    retract_length=1,
-    retract_speed=30,
-    retract_restart_extra=0,
-    retract_before_travel=2,
-    retract_lift=0,
-    retract_layer_change=True,
-    #wipe=False,
-
-    retract_length_toolchange=1,
-    retract_restart_extra_toolchange=1,
-
-    cooling=False,
-    min_fan_speed="35",
-    max_fan_speed="100",
-    bridge_fan_speed="100",
-    fan_below_layer_time=60,
-    slowdown_below_layer_time=30,
-    min_print_speed=10,
-    disable_fan_first_layers=1,
-    fan_always_on=False,
-
-    skirts=1,
-    skirt_distance=6,
-    skirt_height=1,
-    min_skirt_length=0,
-    brim_width=0,
-
-    scale=1, # TODO
-    rotate=0, # TODO
-    duplicate=1,
-    duplicate_grid=(1,1), # TODO ?!
-    duplicate_distance=6,
-    xy_size_compensation=0,
-
-    complete_objects=False,
-    extruder_clearance_radius=20,
-    extruder_clearance_height=20,
-
-    notes="",
-    resolution=0,
-
-    extrusion_width="100%",
-    first_layer_extrusion_width="100%",
-    perimeter_extrusion_width="100%",
-    external_perimeter_extrusion_width="100%",
-    infill_extrusion_width="100%",
-    solid_infill_extrusion_width="100%",
-    top_infill_extrusion_width="100%",
-    support_material_extrusion_width="100%",
-    bridge_flow_ratio=1,
-
-    extruder_offset="0x0",
-    perimeter_extruder=1,
-    infill_extruder=1,
-    support_material_extruder=1,
-    support_material_interface_extruder=1,
-    ooze_prevention=False,
-    standby_temperature_delta=-5
+    nozzle_diameter = 0.5,
+    print_center = (100, 100), # TODO
+    z_offset = 0,
+    gcode_flavor = GcodeFlavors.REPRAP,
+    use_relative_e_distances = 0, 			# 0 (unchecked) or 1 (checked)
+    use_firmware_retraction = 0, 			# 0 (unchecked) or 1 (checked)
+    gcode_arcs = 0, 					# 0 (unchecked) or 1 (checked)
+    gcode_comments = 0, 				# 0 (unchecked) or 1 (checked)
+    vibration_limit = 0,
+
+    filament_diameter = 3,
+    extrusion_multiplier = 1,
+    temperature = 200,
+    first_layer_temperature = 200,
+    bed_temperature = 0,
+
+    travel_speed = 130,
+    perimeter_speed = 30,
+    small_perimeter_speed = 30,
+    external_perimeter_speed = "70%",
+    infill_speed = 60,
+    solid_infill_speed = 60,
+    top_solid_infill_speed = 50,
+    support_material_speed = 60,
+    support_material_interface_speed = "100%",
+    bridge_speed = 60,
+    gap_fill_speed = 20,
+    first_layer_speed = "30%",
+
+    perimeter_acceleration = 0,
+    infill_acceleration = 0,
+    bridge_acceleration = 0,
+    first_layer_acceleration = 0,
+    default_acceleration = 0,
+
+    layer_height = 0.3,
+    first_layer_height = 0.35,
+    infill_every_layers = 1,
+    solid_infill_every_layers = 0,
+
+    perimeters = 3,
+    top_solid_layers = 3,
+    bottom_solid_layers = 3,
+    fill_density = "40%",
```

Comment:
I run a little script in python to compare Slic3r's settings(those are stored in $HOME/.Slic3r for Linux and in user_folder->AppData->Roaming->Slic3r in Windows) and the ones that are in the profile file after uploading it using the plugin that indicated me which settings were in the imported profile but not in Slic3r profiles. Looking at the results, some parameters were added and others (solid_layers is one of them) were removed as they were not part of Slic3r profiles in any of the stable versions. Depending on the version used, there are some parameters unknown by Slic3r that will be ignored, not affecting in the gcode generated.